modules/storage-persistent-storage-azure-file-secret.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="create-azure-file-secret_{context}"]
+= Store the Azure File share credentials in a Secret
+
+{product-title} stores the Azure account and key in a Secret. This Secret
+is then used to access the Azure File share.
+
+.Prerequisites
+
+* An Azure File share exists.
+* The credentials to access this share, specifically the storage account and
+key, are available.
+
+.Procedure
+
+* Create a Secret containing the Azure File credentials, substituting the
```

Comment:
s/containing/that contains

s/credentials, substituting... /credentials: 
Add an annotation to the command.